internal-parallel.h

```diff
@@ -138,11 +136,11 @@ struct piter {
 static void piter_init(struct piter * pi,
         unsigned char * Pmin, unsigned char * Pmax, int Plength,
         struct crstruct * d) {
-    pi->stable = ta_malloc("stable", int, Plength);
-    pi->narrow = ta_malloc("narrow", int, Plength);
+    pi->stable = malloc(Plength * sizeof(int));
+    pi->narrow = malloc(Plength * sizeof(int));
```

Comment:
Should this be malloc not mpiu_malloc?

Reply:
I would rather not going through mpiu_malloc, because Plength is small -- the field doesn't scale with the number of items being sorted. 

We use mymalloc for two reasons: tracking allocation and avoid fragmentation:
short living, small (transient) allocations do not affect our overall understanding of memory budget, and they do not interfere with the fragmentation inside mymalloc's pool when allocated via stdlib.